graphql_compiler/compiler/ir_lowering_match/ir_lowering.py

```diff
@@ -93,95 +93,47 @@ def visitor_fn(expression):
     return new_ir_blocks
 
 
-def lower_string_operators(ir_blocks):
-    """Lower Filter blocks that use the "has_substring", "starts_with", or
-    "ends_with" operation into MATCH-representable form."""
+def prepend_wildcard(expression):
```

Comment:
Any reason to make these functions public, instead of prefixing them with an underscore and therefore signaling that they are meant to be private?

Also, please add docstrings.

Reply:
I didn't know about that convention, thank you